FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -124,6 +124,53 @@ The browser verifies it witnessed a Fetch request to the `seller`’s origin wit
 
 ## Device Orchestrated Multi-Seller Auctions
 
+Bidding and Auction Services auctions can also be used as component auctions. It
+is possible for multiple Bidding and Auction Services auctions to be used as
+component auctions for an on-device top-level auction.
+
+For efficiency, a top-level seller can construct encrypted requests for multiple
+component sellers in a single call to `navigator.getInterestGroupAdAuctionData()`:
+
+```javascript
+const results = await navigator.getInterestGroupAdAuctionData({
+  'sellers': [
+    {
+      'seller': 'https://www.component-ssp1.example.com',
+      'coordinatorOrigin': 'https://publickeyservice.pa.gcp.privacysandboxservices.com/',
+    },
+    {
+      'seller': 'https://www.component-ssp2.example.com',
+      'coordinatorOrigin': 'https://publickeyservice.pa.gcp.privacysandboxservices.com/',
+  }],
+  // 'requestSize' the affects the size of the returned request (optional).
+  'requestSize': 51200,
+  // 'perBuyerConfig' specifies per-buyer options for size optimizations when
+  // constructing the blob (optional).
+  'perBuyerConfig': {
+    'https://buyer1.origin.example.com': {
+      // 'targetSize' specifies the size of the blob to devote to this buyer
+      // (optional).
+      'targetSize': 8192,
+    },
+    'https://buyer2.origin.example.com': {}
+  }
+});
+
+const requestId = result.requestId;
```

Comment:
Still using result.

Note that Chrome balks at syntax here:

```
const requests = {results.requests[0].seller: results.requests[0].request,
                  results.requests[1].seller: results.requests[1].request};
```

FWIW, here's what I replaced it with when testing:

```
            const requestId = results.requestId;
            const sellerRequests = {};
            for( const sellerResult of results.requests) {
                const sellerOrigin = sellerResult.seller;
                sellerRequests[sellerOrigin] = {request, error} = sellerResult;
                if(sellerResult.error) {
                    console.log("Seller '%s' auctionData error: %s", sellerOrigin, sellerResult.error);
                }
            }
```

Reply:
Thanks. I fixed the example.